lib/jobba.rb

```diff
@@ -38,4 +38,12 @@ def self.redis
     )
   end
 
+  # Clears the whole shebang!  USE WITH CARE!
+  def self.clear_all_jobba_data!
+    keys = Jobba.redis.keys("*")
```

Comment:
I moved it here out of `spec_utils` because I wanted to use it more easily from the console (still for testing)